src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -534,9 +534,9 @@ export default {
 		// check for applied promo
 		this.verifyOrApplyPromotion();
 		// update basket state if any loans are already in the basket
-		if (this.itemsInBasket.length) {
-			this.updateBasketState();
-		}
+		// if (this.itemsInBasket.length) {
+		// 	this.updateBasketState();
+		// }
```

Comment:
experimenting with removing this completely. Pretty sure it's a redundant update at this point